plugins/ProcessBase.h

```diff
@@ -45,6 +45,9 @@ class ProcessBase : public Host {
 
 	procptr_t peekPtr(const procptr_t address) const;
 
+	/// Resolves x64's RIP relative addressing.
+	procptr_t peekRIP(const procptr_t address) const { return address + peek< uint32_t >(address) + 4; }
```

Comment:
Where does the `+4` come from and what does it do? This smells a bit like a magic number that should instead be a named constant in order for it to have a meaningful name.

Reply:
https://stackoverflow.com/a/46648749